buildSrc/src/main/kotlin/kotest-script.gradle.kts

```diff
@@ -10,6 +10,6 @@ dependencies {
     testImplementation("io.kotest:kotest-property:$kotestVersion")
 }
 
-tasks.test {
+tasks.withType<Test> {
```

Comment:
I would leave this as `test`, as we do not want to use the junit platform for all kinds of tests in the future.